src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -706,47 +710,40 @@ void AppHost::_GlobalHotkeyPressed(const long hotkeyIndex)
     {
         return;
     }
-    // Lookup the matching keychord
-    Control::KeyChord kc = _hotkeys.at(hotkeyIndex);
-    // Get the stored Command for that chord
-    if (const auto& cmd{ _hotkeyActions.Lookup(kc) })
-    {
-        if (const auto& summonArgs{ cmd.ActionAndArgs().Args().try_as<Settings::Model::GlobalSummonArgs>() })
-        {
-            Remoting::SummonWindowSelectionArgs args{ summonArgs.Name() };
 
-            // desktop:any - MoveToCurrentDesktop=false, OnCurrentDesktop=false
-            // desktop:toCurrent - MoveToCurrentDesktop=true, OnCurrentDesktop=false
-            // desktop:onCurrent - MoveToCurrentDesktop=false, OnCurrentDesktop=true
-            args.OnCurrentDesktop(summonArgs.Desktop() == Settings::Model::DesktopBehavior::OnCurrent);
-            args.SummonBehavior().MoveToCurrentDesktop(summonArgs.Desktop() == Settings::Model::DesktopBehavior::ToCurrent);
-            args.SummonBehavior().ToggleVisibility(summonArgs.ToggleVisibility());
-            args.SummonBehavior().DropdownDuration(summonArgs.DropdownDuration());
+    const auto& summonArgs = til::at(_hotkeys, hotkeyIndex);

```

Comment:
As you can see here, a lot of casting is gone without the `_hotkeyActions` map.
Otherwise this code is still the same - just less indented.